SR2021: Issue 03 (#11)
srobo/srawn: SR2021/2020-02-28-srawn-03.md

The Student Robotics (Almost) Weekly Newsletter is a (roughly) fortnightly newsletter which summarises the goings‐on across Student Robotics. **Emboldened** items are ones for which additional help has been requested.

# Headlines

### Mentoring

**There are a number of teams who need mentors to help them with this year's game.** No experience with mentoring or the simulator required, but experience with SR is definitely useful!

If you're interested, please let the competition team know!

> ### Module III League

Reviewer: Would be super awesome to use the Roman numeral here (and elsewhere) when referring to the modules:
```suggestion
### Module Ⅲ League
```
The league sessions however are numbered using Arabic numerals.

Author: TIL that these are included in unicode!

> We announced Module III at the league session, and teams have 6 weeks to react until the first league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.

Reviewer: Maybe remove "first"? It's a little confusing as this won't be the first league session -- it's going to be the third (overall).
```suggestion
We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```

Author: Yeah, absolutely agree. Looks like this content was copy-pasted from a previous issue when it was submitted.

> We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.

Reviewer: ```suggestion
We announced Module Ⅲ at the last league session, and teams have 6 weeks to react until the next league session. See [the announcement](https://studentrobotics.org/news/2021-02-06-sr2021-second-league-sessions) for more info.
```
(slightly surprised that this link was working without this)

League session 3 is in just few weeks on 20th March, where teams will compete in the game defined by Module Ⅲ. Here we'll also unveil Module Ⅳ, the final module of SR2021, and what will be played in league session 4 and the knockouts.

### Competition Doing: 6th March

We'll be holding "Doing" on Saturday 6th of March from 2PM UK time to 6PM (but may last longer), working on various competition topics, such as updates to the competition software, wrapping up any changes to Module Ⅳ, and other smaller simulator fixes.

## Competition

> ### Module IV Design

Reviewer: ```suggestion
### Module Ⅳ Design
```
